### DISCUSSION TITLE
JCCI 的openapi问题

### DISCUSSION THREAD

**Discussion by leefuqing01** [baikaishuipp/jcci#27] [General]
JCCI 这个工具非常好用，基于此可以分析变更的影响面。

1、在实际使用过程中发现最终生成的.CCI 文件默认在隐藏文件件中，是否可以通过指定存储目录的形式，重新定义文件夹的修正目录。

**Comment by baikaishuipp** [comment]
可以在运行时配置项目clone地址，cci文件都在项目目录内，示例代码：
import jcci.config as config
config.project_path = 'D:/pycharm/github/jcci'
config.db_path = 'D:/pycharm/github/jcci'
